webpack/loader.d.ts

```diff
@@ -0,0 +1,6 @@
+declare function style9Loader(
+  input: string,
+  inputSourceMap?: object | null
```

Comment:
Seems like this should be `object | boolean`: https://babeljs.io/docs/en/options#inputsourcemap

Reply:
> Seems like this should be `object | boolean`: https://babeljs.io/docs/en/options#inputsourcemap

Huh; then there seems to be an inconsistency between the docs (`object | boolean`), the [DefinitelyTyped declarations](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/97784d59abc06f44cf402d0dff702d7bef2e9cbf/types/babel-core/index.d.ts#L102) in the RT repo (`object`), and [the published version of `@types/babel__core`](https://unpkg.com/@types/babel__core@7.1.9/index.d.ts) (line 200: `object | null`).

The docs are probably the more reliable source of truth, though, so I'll make the change.